src/js/main.js

```diff
@@ -187,20 +187,21 @@ class Site {
       const age = dayjs().diff(birthday, 'years');
       if (age >= 21) {
         Cookies.set('legalAge', true, { expires: 1 }); // Expires in 1 day
-        $('body').addClass('legal-age');
+        $('body').removeClass('age-check');
       } else {
         console.log("not of age");
       }
     });
   }
 
   checkForCookie() {
+    console.log("checking for cookie");
     const cookie = Cookies.get('legalAge');
     console.log(cookie);
     if (cookie) {
-      $('body').addClass('legal-age');
+      console.log("your old enough");
```

Comment:
since we aren't doing anything when the cookie exists, the entire condition can be: 
```
if (!cookie) {
  $('body').addClass('age-check');
}
```
and we don't need an `else`